bin/cigar

```diff
@@ -27,14 +27,16 @@ if (isset($options['help'])) {
   cigar [options]
 
 \033[33mOptions:\033[0m
-      \033[32m-c file.json, --config=file.json\033[0m   Use the specified config file instead of the default .cigar.json file 
-      \033[32m-u URL,       --url=URL\033[0m            Base URL for checks, e.g. https://example.org/
-      \033[32m-i,           --insecure\033[0m           Allow invalid SSL certificates
-      \033[32m-a,           --auth\033[0m               Authorization header "\074type\076 \074credentials\076"
-      \033[32m-h,           --header\033[0m             Custom header "\074name\076: \074value\076"
-      \033[32m-j,           --json\033[0m               Output JSON
-      \033[32m              --quiet\033[0m              Do not output any message
-      \033[32m              --version\033[0m            Print the version of Cigar 
+      \033[32m-c file.json, --config=file.json\033[0m         Use the specified config file instead of the default .cigar.json file 
+      \033[32m-u URL,       --url=URL\033[0m                  Base URL for checks, e.g. https://example.org/
+      \033[32m-i,           --insecure\033[0m                 Allow invalid SSL certificates
+      \033[32m-a,           --auth\033[0m                     Authorization header "\074type\076 \074credentials\076"
+      \033[32m-h,           --header\033[0m                   Custom header "\074name\076: \074value\076"
+      \033[32m-ct,          --connect-timeout=TIMEOUT\033[0m  Connect Timeout
```

Comment:
Help output will need changing with the change of the `ct` option

Reply:
Removed the mention of `ct` here